src/lib.rs

```diff
@@ -4,8 +4,12 @@
 #![cfg_attr(test, deny(warnings))]
 
 extern crate clap;
+extern crate roff;
+
+mod man;
 
 use clap::{App, Arg, ArgSettings};
+pub use man::*;
```

Comment:
I'd only expose some very carefully selected items.

Reply:
I think if we can get https://github.com/rust-clique/man/issues/9 to land, this will also be solved. For now it mostly enforces the same behavior we'd have as putting everything in `lib/` (e.g. less typing until we clean things up, haha).